### DISCUSSION TITLE
Hello, the squares didnt draw every x and y client using socket.io and simple-json-database.

### DISCUSSION THREAD

**Discussion by DrFrankatron** [socketio/socket.io#4958] [Q&A]
Hello, the squares didnt draw every x and y client using socket.io and simple-json-database. May it disapeared? Here is my node.js files:
[test20.zip](https://github.com/socketio/socket.io/files/14390532/test20.zip)
Apart of that, the entity table doesnt show all objects. Look at db1.json and start.bat outputs:
![image](https://github.com/socketio/socket.io/assets/157644009/8c8d512c-5db7-4fbe-8f47-67d43ec9e484)
[db1.json](https://github.com/socketio/socket.io/files/14390549/db1.json)

**Comment by darrachequesne** [comment]
Hi! Could you please explain what you are trying to achieve?

Note: this does not seem to be specifically related to Socket.IO, you might have more chance on [StackOverflow](https://stackoverflow.com/) for this kind of questions.
